kotlin/src/main/java/app/rive/runtime/kotlin/RiveArtboardRenderer.kt

```diff
@@ -154,19 +166,20 @@ open class RiveArtboardRenderer(
         loop: Loop = Loop.AUTO,
         direction: Direction = Direction.AUTO,
         isStateMachine: Boolean = false,
+        settleInitialState: Boolean = true,
```

Comment:
Oh, we're defaulting to true? Is that so our users get "what they expect"?

Reply:
Fair fair, yeah, bit of a shame that we're sprinkling our own code with `settleInitialState = false`, but this is going to make it so much easier for our users.